systemd/opensuse/openqa-continuous-update.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Trigger of continuous openQA deployments, see https://progress.opensuse.org/issues/105379
+
+[Timer]
+OnBootSec=10min
```

Comment:
https://progress.opensuse.org/issues/105379#Suggestions says 5 minutes. I would be ok to accept 5 minutes as continuous, but 10 minutes? I don't know ;) But seriously, I would prefer 5 minutes. Any good reason against?

Reply:
No, I have nothing against 5 minutes. It'll usually just exiting very quickly anyways. I suppose if an update takes longer than 5 minutes the systemd timer will be smart enough to handle it.

Reply:
Zypper would fail when trying to get the lock